build/openwisp_base/Dockerfile

```diff
@@ -20,12 +20,12 @@ ENV PYTHONPATH=/install/lib/python3.7/site-packages
 # Python Packages
 RUN pip install --editable git+git://github.com/openwisp/openwisp-controller#egg=openwisp_controller \
                 --prefix='/install'
+RUN pip install --upgrade https://github.com/openwisp/django-freeradius/tarball/master \
```

Comment:
Was this change intentional?  Just curious if django-freeradius needed to be installed before openwisp-radius.

Reply:
Yeah it's an [ongoing issue](https://github.com/openwisp/openwisp-radius/issues/73), just a temporary fix.